.devcontainer/devcontainer.json

```diff
@@ -9,6 +9,7 @@
   "mounts": [
     "type=volume,source=zmk-root-user,target=/root",
     "type=volume,source=zmk-config,target=/workspaces/zmk-config",
+    "type=volume,source=zmk-modules,target=/workspaces/zmk-modules",
```

Comment:
Just a note: approving this change, from a container setup perspective. Will let the docs team handle final approval/merge of this PR though. Thanks!